core/src/main/java/io/questdb/cairo/TableWriter.java

```diff
@@ -3828,7 +3906,6 @@ private void o3MergeIntoLag(long mergedTimestamps, long countInLag, long mappedR
                                 mappedRoHi,
                                 ColumnType.isVariableLength(type) ? o3MergeLagVarColumnRef : o3MergeLagFixColumnRef
                         );
-                        o3PendingCallbackTasks.add(task);
                     } finally {
```

Comment:
Do we really need this `try/finally` block here? I don't think anything in the `try` block may throw. The same question applies to other similar places where we initialize `O3CallbackTask`s.

Reply:
removed